corehq/blobs/__init__.py

```diff
@@ -19,6 +19,11 @@ def get_blob_db():
     return _db[-1]
 
 
+def get_blob_db_exporter(slug, domain):
+    from .migratingdb import MigratingBlobDB
+    return MigratingBlobDB(_get_zip_db(slug, domain), get_blob_db())
```

Comment:
https://github.com/dimagi/commcare-hq/pull/13478/commits/55f06adcea7fa97659a32b0dde38e5e52a4abf06 & https://github.com/dimagi/commcare-hq/pull/13478/commits/b8ff4f485d46c608e28a6e71a8678797c78a6fde

I'm fine with either approach, but I'm sure why one would be better than hte other. I originally used `MigratingBlobDB` because you need access to both the zipfile "db" and the current riak instance. It does seem weird because we're not "migrating" though.